### PR TITLE
global: sqlalchemy relationships refactor

### PR DESCRIPTION
**Pull request by krzysztof**
* Fixed the backref relationships in models.

Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

**Comment by lnielsen** [comment]
Which problems does this fix solve?

**Comment by krzysztof** [comment]
None, just convenience of having consistent ORM definitions across modules.

File: tests/__init__.py [hunk 1]
```diff
@@ -16,3 +16,4 @@
 # You should have received a copy of the GNU General Public License
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+"""OAuthClient unit tests."""
```
Comment (jirikuncar) [inline tests/__init__.py]: We should remove `__init__.py` file from `tests`.

**Comment by jirikuncar** [comment]
@krzysztof `s/refactor/refactoring/`, `s/Fixed/Fixes/`

File: tests/__init__.py (hunk 1 above)
Comment (krzysztof) [inline tests/__init__.py]: Actually it's needed for importing ``tests/helpers.py``: https://github.com/inveniosoftware/invenio-oauthclient/blob/master/tests/test_contrib_cern.py#L32

Comment (jirikuncar) [inline tests/__init__.py]: `from helper import ...` will work without `__init__.py`

**Pull request merged**